repository/Signals.package/monticello.meta/version

```diff
@@ -1 +0,0 @@

Note: That trick is not possible or wanted for other processes because other processes do not have a default shared queue that is processed frequently.' id '27595dae-6d92-354a-807b-46fd5512ba3b' date '4 June 2010' time '1:12:23.656 pm' author 'mt' ancestors ((name 'Signals-mt.5' message '- signal spy added to support convenient signal testing' id '6d9c309c-f595-ad45-b041-7981310fad57' date '3 June 2010' time '12:25:19.815 pm' author 'mt' ancestors ((name 'Signals-mt.4' message '- queued connections can now be blocking, i.e., sender waits until the delivery is successful
- see test10 for an example' id '2b395019-2798-e843-ba7a-5d273c37a7cb' date '3 June 2010' time '10:39:53.516 am' author 'mt' ancestors ((name 'Signals-mt.3' message '- queued connections are thread-safe now' id '1ca001d0-a4bd-da42-8f28-a33bfaea1d3c' date '3 June 2010' time '8:52:05.059 am' author 'mt' ancestors ((name 'Signals-mt.2' message '- queued connections added to support process synchronization points (= queues)
- see test08 and test09 for examples

! thread-safe connection data structures needed for a safe emitting of signals out of any process !' id '8696f5d9-684f-d24d-80de-52fc3b08ebd3' date '3 June 2010' time '8:30:31.668 am' author 'mt' ancestors ((name 'Signals-mt.1' message 'Simple synchronous event/callback mechanism inspired by Qt. Convenient connection between signal-methods and any target-methods possible. Supports argument shadowing/discarding and reorderung when delivering a signal to a receiver.

- easy signal overview per class because signals are methods
- no clumsy event object necessary to be processed
- signal processing methods are just normal methods (not special #update: as needed with #changed)

See the tests for simple examples.' id '442bd946-6de3-694d-a6d9-b0b606309ebd' date '2 June 2010' time '10:53:54.289 pm' author 'mt' ancestors () stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())) stepChildren ())
```

Comment:
Am, there were no contribution instructions in the readme. Did I do anything wrong by committing my changes using the Git Browser?